src/utils/setProductConfiguration.js

```diff
@@ -16,6 +16,10 @@ function setProductConfiguration(config) {
   this.state.unbxdCore.setVariantsCount(variantsCount);
   this.state.unbxdCore.setVariantAttributes(variantRequiredFields);
   this.state.unbxdCore.setVariantsGroupBy(groupBy);
+  
+  paginationType === paginationTypes.INFINITE_SCROLL? this.state.unbxdCore.options.pagination.type = paginationTypes.INFINITE_SCROLL :
```

Comment:
Can we do the same with an if block and an OR condition.

Reply:
ternary operators are preferred over if and else generally..

Reply:
I think readable code is more important.

Reply:
it is readable.. Only when u need to perform more than one assignment should you go for if else block.
But here it is just one assignment

Reply:
Only when one wants to perform more than one assignment should one go for if else block..

Reply:
which version do you think is easier to read.

```
if(paginationType===paginationTypes.INFINITE_SCROLL ||
paginationType===paginationTypes. CLICK_N_SCROLL ||
paginationType===paginationTypes. FIXED_PAGINATION){
    this.state.unbxdCore.options.pagination.type = paginationType;
} else {
    this.state.unbxdCore.options.pagination.type = paginationTypes. FIXED_PAGINATION;
}
```

OR

```
paginationType === paginationTypes.INFINITE_SCROLL? this.state.unbxdCore.options.pagination.type = paginationTypes.INFINITE_SCROLL :
  paginationType === paginationTypes.CLICK_N_SCROLL ? this.state.unbxdCore.options.pagination.type = paginationTypes.CLICK_N_SCROLL:
  this.state.unbxdCore.options.pagination.type = paginationTypes.FIXED_PAGINATION

```

Reply:
done